java-frontend/src/main/java/org/sonar/java/ast/JavaAstScanner.java

```diff
@@ -54,29 +52,13 @@ public JavaAstScanner(ActionParser<Tree> parser, @Nullable SonarComponents sonar
   }
 
   public void scan(Iterable<File> files) {
-    ProgressReport progressReport = new ProgressReport("Report about progress of Java AST analyzer", TimeUnit.SECONDS.toMillis(10));
-    progressReport.start(Lists.newArrayList(files));
-
-    boolean successfullyCompleted = false;
-    boolean cancelled = false;
-    try {
-      for (File file : files) {
-        if (analysisCancelled()) {
-          cancelled = true;
-          break;
-        }
-        simpleScan(file);
-        progressReport.nextFile();
+    for (File file : files) {
+      if (analysisCancelled()) {
+        break;
       }
-      successfullyCompleted = !cancelled;
-    } finally {
-      if (successfullyCompleted) {
-        progressReport.stop();
-      } else {
-        progressReport.cancel();
-      }
-      visitor.endOfAnalysis();
-    }
+      simpleScan(file);
+   }
+   visitor.endOfAnalysis();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)